### PR TITLE
{bio}[foss/2020a] AFNI v21.3.10 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by kErica**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@kErica we try to stick to a single version of dependencies per toolchain version - there are exceptions, and in the case of R you should be able to make use of the exception by adding `-R-%(rver)s` to the `versionsuffix`

**Comment by kErica** [comment]
Thank @migueldiascosta for the suggestion, I wasn't sure if to add it or not

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/ab90c366e3d2689180022da3ec3a70e9 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/f36ce445b38c08bab511e6db8f470d30 for a full test report.

File: easybuild/easyconfigs/a/AFNI/AFNI-21.11.07-foss-2020a-R-3.6.3-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+versionsuffix = '-R-%(rver)s-Python-%(pyver)s'
+
+source_urls = ['https://afni.nimh.nih.gov/pub/dist/tgz/']
+sources = [{
+    'download_filename': '%(namelower)s_src.tgz',
+    'filename': SOURCELOWER_TGZ
+}]
+patches = ['%(name)s-%(version)s-Makefile.patch']
+checksums = [
+    '955e703d751c0179be2a3d88f46ee03b0e5c6619fbeea1617ec7afd2e8a31e16',  # afni-21.11.07.tgz
```
Comment (verdurin) [inline easybuild/easyconfigs/a/AFNI/AFNI-21.11.07-foss-2020a-R-3.6.3-Python-3.8.2.eb:28]: @kErica could you check the checksum please?
On several systems, I see `c436920f5a1c5fc019f463773d0ce65c7763b3a686a28ff73319a2dabfc17adf`

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/a/AFNI/AFNI-21.11.07-foss-2020a-R-3.6.3-Python-3.8.2.eb (hunk 1 above)
Comment (kErica) [reply easybuild/easyconfigs/a/AFNI/AFNI-21.11.07-foss-2020a-R-3.6.3-Python-3.8.2.eb]: I checked it. It seems the developper changes the source file with the name I used regularly - no versioning of the sourcefile.
I change the source url and version to the one they have on their github project - more reproducible

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@kErica here is the error from `generoso`:
```
gcc -O2 -ftree-vectorize -march=native -fno-math-errno -fopenmp -fPIC -DREAD_WRITE_64 -DLINUX2  -DWHATS_MY_EXEPATH -o afni afni.o afni_func.o afni_widg.o afni_warp.o afni_fimmer.o afni_slice_short.o afni_slice_float.o afni_filer.o afni_slice_byte.o  afni_slice_complex.o afni_pcor.o afni_pcor_float.o afni_transforms.o pbar.o afni_graph.o afni_plugin.o afni_cluster.o parser.o parser_int.o afni_plugout.o afni_fimfunc.o afni_setup.o afni_receive.o mcw_graf.o afni_splash.o afni_pplug_env.o afni_pplug_2dfunc.o afni_friends.o afni_ttren.o afni_pplug_1dfunc.o afni_driver.o afni_niml.o afni_sumafunc.o afni_slice_rgbyte.o afni_version.o afni_lock.o afni_vol2surf.o afni_pplug_instacorr.o afni_pplug_instacalc.o afni_pplug_tstat.o afni_whelp.o whats_my_exepath.o  -rdynamic -L. -L.  -lmrix -lcoxplot -lmri  libXmHTML.a libjpeg.a -lmri -lf2c -lXm -lXt -lXft -lXp -lXpm -lXext -lXmu -lSM -lICE -lX11  -lpng -ljpeg -lz -lexpat -lm  -ldl -lc -lm -lc
./libmri.a(thd_niml.o):thd_niml.c:function read_niml_file: error: undefined reference to 'STATUSs'
libXmHTML.a(LZWStream.o):LZWStream.c:function LZWStreamInit: warning: the use of `tmpnam' is dangerous, better use `mkstemp'
./libf2c.a(open.o):open.c:function f_open: warning: the use of `tmpnam' is dangerous, better use `mkstemp'
./libf2c.a(endfile.o):endfile.c:function t_runc: warning: the use of `tmpnam' is dangerous, better use `mkstemp'
collect2: error: ld returned 1 exit status
make: *** [Makefile.INCLUDE:1841: afni] Error 1
```

**Comment by boegel** [comment]
closing this since `foss/2020a` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**